### PR TITLE
Update environment-scaling.md

### PR DESCRIPTION
**Pull request by ThomasLehnerSpryker**
explained how a load test can look like

## PR labels

When the PR is ready for review, add a **TW review needed** label. This lets us keep track of PRs that need to be merged and merge them in time.

## PR Description
Describe the context for your changes and the changes you've made.

## Tickets

If changes are associated with a ticket, add a docs ticket here. 

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/ca/dev/environment-scaling.md [hunk 1]
```diff
@@ -33,7 +33,29 @@ There is an autoscaling configuration that has a service-level CPU threshold con
...
+
+Due to the inherent delay in horizontal scaling on application tasks, significant load spikes should generally be avoided. While testing with high RPMs without "warmup" (gradually ramping up the traffic over several minutes to allow step scaling to deploy now tasks) is generally a good way to test whether the autoscale guardrails are set correctly, the application will need to be prepared on our side for this testing scenario - we will temporary deploy the maximum count of tasks configured for your environment. You can mention this as a comment in your Load Event Announcement. Without it, you will quickly see timeouts during your load test if your load test is significant enough to overwhelm the vertical scaling buffers configured for your environment.
```
Comment (andriitserkovnyi) [inline docs/ca/dev/environment-scaling.md:38]: replace RPM with the full form

Comment (andriitserkovnyi) [reply docs/ca/dev/environment-scaling.md:38]: Are significant spikes avoided because of the delay or should I do something to avoid them?

Comment (ThomasLehnerSpryker) [reply docs/ca/dev/environment-scaling.md]: adjusted, clearer now?

Comment (andriitserkovnyi) [reply docs/ca/dev/environment-scaling.md]: It's still not clear what to do with the information about spikes. Is this something the reader needs to do or is this a general practice to avoid them by, for example, by staggering email campaigns?
